configuration/parse.go

```diff
@@ -260,14 +236,44 @@ func determineIfLineIsExclusion(ogLine string, config *Configuration) error {
 		if until != nil {
 			parseDate, err := time.Parse("2006-01-02", strings.TrimSpace(until[2]))
 			if err != nil {
-				return errors.New(fmt.Sprintf("failed to parse until at line '%s'. Expected format is 'until=yyyy-MM-dd'", ogLine))
+				return fmt.Errorf("failed to parse until at line '%s'. Expected format is 'until=yyyy-MM-dd'", ogLine)
 			}
 			if parseDate.After(time.Now()) {
-				config.CveList.Cves = append(config.CveList.Cves, cveOnly)
+				c.CveList.Cves = append(c.CveList.Cves, cveOnly)
 			}
 		} else {
-			config.CveList.Cves = append(config.CveList.Cves, cveOnly)
+			c.CveList.Cves = append(c.CveList.Cves, cveOnly)
+		}
+	}
+	return nil
+}
+
+func (c *Config) handleOssIndexOutputFormat(outputFormat string) error {
+	setTextOutputFormat := func() {
+		OssIndexOutputFormats["text"].(*audit.AuditLogTextFormatter).Quiet = &c.Quiet
+		OssIndexOutputFormats["text"].(*audit.AuditLogTextFormatter).NoColor = &c.NoColor
+	}
+
+	if OssIndexOutputFormats[outputFormat] != nil {
+		switch outputFormat {
+		case "text":
+			setTextOutputFormat()
+		case "csv":
+			OssIndexOutputFormats[outputFormat].(*audit.CsvFormatter).Quiet = &c.Quiet
+		default:
+			setTextOutputFormat()
 		}
```

Comment:
I dont understand the default case here?? If it not text...we set the text output formats?? So the only cases we would hit the default would be "json" and "json-pretty"??

Reply:
I was flying way to close to the sun, Icarus style: bee0c6f